lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.0-preview.0",
+  "version": "3.0.0-preview.1",
```

Comment:
For easier review, you can see just the changes related to bumping up the version. To see such subset (the last X commits from me): https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1245/files/dccf1637111c700a347ba6d001cab24454a14db2..08aa947d94f74e551e5cc194c4fdec935d2aeb1f